modules/HomeKitGate/index.js

```diff
@@ -115,10 +115,15 @@ HomeKitGate.prototype.init = function (config) {
 			});
 		}
 		else if (deviceType == "switchMultilevel") {
-			var serviceUUID = "1004";
-		
-			var service = accessory.addService(serviceUUID, "Multilevel Switch");
-			
+			var serviceUUID = HomeKit.Services.Lightbulb;
+
+                        var service = accessory.addService(serviceUUID, title + " Dimmer");
+
+                        m.level = service.addCharacteristic(HomeKit.Characteristics.PowerState, "bool", {
```

Comment:
This change is broken. This sets m.level twice. It needs to set another property so both characteristics can be notified when the level changes. As is, the PowerState won't be updated on your iOS devices when the level changes.

I have made this change and will clean up my changes and make a pull request.